blender/arm/logicnode/material/LN_set_material_image_param.py

```diff
@@ -1,17 +1,43 @@
 from arm.logicnode.arm_nodes import *
 
 class SetMaterialImageParamNode(ArmLogicTreeNode):
-    """TO DO."""
+    """Set an image value material parameter to the specified object. 
+    
+    @seeNode Get Scene Root
+    
+    @input Object: Object whose material parameter should change. Use `Get Scene Root` node to set paramter globally.
```

Comment:
Great work 👍  Is "Use `Get Scene Root` node to set paramter globally" a leftover from older experiments? It looks like unchecking `Per Object` alone is sufficient to set the parameter globally. Also, there is a small typo in "paramter".

Reply:
Thanks!

That get scene root node is added intentionally. We need a way to remove this object from the map when we want it to follow global values again. For example, consider an objects A using the same material as others.

Say we need to set A's parameter only, then we enable `per object` and set the property. This adds the object to the map. 

Now, to remove the object from this map, and revert back to global value, we disable `per object`. 

But we also need a way to set the global parameter without removing this object from the map. 

For this, we use scene root object. This way, we do not affect any objects with their own parameters, but directly modify the global parameter.

Also thanks for pointing out the typo, will make a commit for that!